### DISCUSSION TITLE
Grafana plugin?

### DISCUSSION THREAD

**Discussion by helmut72** [openobserve/openobserve#1339] [Q&A] [answered]
Hi,

I test OpenObserve database with Fluentbit on my Raspberry and it looks good so far. Log handling is better than with Grafana, but I miss all the other features of Grafana:

* a lot of different visualization panels like maps and also 3rd party panels
* OIDC and LDAP in the free/Open Source version
* a lot of datasources plugins
* mobile view

When testing Zincsearch, I was able to connecting Grafana to Zincsearch with Elastic datasource, but it looks like it's now impossible in the free version of OpenObserve:
https://openobserve.ai/docs/zplane/

Then I found Grafana datasource:
https://openobserve.ai/docs/operator-guide/grafana_plugin/

Which reads it's also an enterprise plugin. But I was able to install it. What will you do in the future? Can it be only installed because it's an early version?

Thank you.

**Comment by prabhatsharma** [comment, marked as answer]
You are free to install and use the grafana plugin for testing, personal and non-commercial use.
